src/compiler/compile/render_dom/wrappers/Element/Attribute.ts

```diff
@@ -65,17 +66,33 @@ export default class AttributeWrapper extends BaseAttributeWrapper {
 				handle_select_value_binding(this, node.dependencies);
 			}
 		}
+		
+		const namespace = this.parent.node.namespace;
+		// some processing only applies to html namespace (and not MathML, SVG, or Svelte Native etc)
+		if (namespace && namespace != 'html' && namespace != namespaces.html) {
+			// attributes outside of the html namespace may be case sensitive
+			// namespaces for which we don't have case corrections, are left in their original case (required for svelte-native)
+			this.name = (valid_namespaces.indexOf(namespace) >= 0) ? fix_attribute_casing(this.node.name) : this.node.name;
```

Comment:
it seemed this is not just about the attribute name case, also set all the metadata to null right?

Reply:
I changed this branch to only apply to unknown namespaces (for safety) and restored the metadata